verify.sh

```diff
@@ -41,6 +41,8 @@ function verify_libbpf() {
         echo "no libbpf package found"
         exit 1
     fi
+    sudo apt-get install -y mlocate
+    locate libbpf
```

Comment:
Won't `dpkg -L libbpf-dev` work?

Reply:
per [this](https://forums.veeam.com/vmware-vsphere-f24/ubuntu-22-04-mlocate-t81712.html) message, `mlocate` is gone in newer ubuntu